build/package.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+	<metadata>
+		<id></id>
+		<version>0.0.0</version>
+		<title></title>
+		<authors></authors>
+		<owners></owners>
+		<projectUrl></projectUrl>
+		<iconUrl></iconUrl>
+		<requireLicenseAcceptance></requireLicenseAcceptance>
+		<description></description>
+		<summary></summary>
+		<copyright></copyright>
+		<licenseUrl></licenseUrl>
+		<language></language>
+		<tags></tags>
+		<dependencies>
+			<dependency id="UmbracoCms" version="[7.8,7.11)" />
```

Comment:
I'd set this to be `[7.8,8)`, so that it includes all v7.8 up to v8, rather than stopping at v7.10.x last patch release.